src/zodbbrowser/browser.py

```diff
@@ -31,6 +33,8 @@
 
 log = logging.getLogger("zodbbrowser")
 
+__version__ = pkg_resources.get_distribution('zodbbrowser').version
```

Comment:
This is also separate PR material, mostly because I need to be convinced it's a good idea.